.github/workflows/update-dotnet-sdk.yml

```
@@ -12,7 +12,7 @@ jobs:
   update-sdk:
     uses: martincostello/update-dotnet-sdk/.github/workflows/update-dotnet-sdk.yml@ff9a6f247704ef1b77eb3c478070d8fa69c2360e # v2.2.3
     with:
-      include-nuget-packages: "Microsoft.AspNetCore.,Microsoft.NET.Test.Sdk"
+      include-nuget-packages: "Microsoft.AspNetCore.,Microsoft.Extensions.,Microsoft.NET.Test.Sdk"
```

When https://github.com/martincostello/update-dotnet-sdk/issues/496 is implemented change to:

```suggestion
      include-nuget-packages: "Microsoft.AspNetCore.,Microsoft.Extensions.,Microsoft.NET.Test.Sdk"
      exclude-nuget-packages: "Microsoft.Extensions.DependencyInjection.Abstractions,Microsoft.Extensions.Logging.Abstractions"
```